### ISSUE TITLE
Publication date potentially corrected from the consolidation service seems to be lost in a limbo 

### ISSUE THREAD

**Issue by lfoppiano** [kermitt2/grobid#760]
Related to #759. 
I tried to solve the problem by enabling the consolidation, however, the correct publication date was not retrieved correctly. 

The BiblioItem object from the consolidation service had: 
 - publicationDate = null 
 - normalisedPublicationDate with the correct value

(I wonder if this is the correct behaviour)

however, in the BiblioItem, after the `correct(...)` method is invoked, the normalisedPublication date is overridden with the first raw publication date: 

```java
// normalization of dates
                if (resHeader != null) {
                    if (resHeader.getPublicationDate() != null) {
                        List<Date> dates = parsers.getDateParser().processing(resHeader.getPublicationDate());
                        // most basic heuristic, we take the first date - to be
                        // revised...
                        if (dates != null) {
                            if (dates.size() > 0) {
                                resHeader.setNormalizedPublicationDate(dates.get(0));
                            }
                        }
                    }
```

This will inevitably lose the good normalised date from the consolidation service. I'm not sure what is the correct behaviour, I supply this issue mostly for information as I assume there many different use cases 😉

**Comment by kermitt2** [comment]
Good catch Luca !! 
It's a bug yes, we should not normalize the publication date if `(resHeader.getNormalizedPublicationDate() != null)` already. 

> (I wonder if this is the correct behaviour)

yes it's the correct behavior, crossref record has only a normalized publication date (no "raw" publication date).

**Comment by lfoppiano** [comment]
Just for reference, this problem is a bit wider than just in random documents. I have more cases, it seems generally due to document segmentation but happening also with the old pdfalto version. 
With the old pdfalto version on Mac and consolidation enabled, this document: 

https://aip.scitation.org/doi/10.1063/1.3238557 

returns: 

```
<date type="published" when="1036010370">10360 10370 10380 10390 10400</date>
```

when processing fulltext.
